docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -0,0 +1,100 @@
+= Integrating CMOS with an existing monitoring pipeline
+:icons: font
+:imagesdir: ../assets/images
+ifdef::env-github[]
+:imagesdir: https://github.com/couchbaselabs/observability/raw/main/docs/modules/ROOT/assets/images
+endif::[]
+
+[abstract]
+== Introduction
+Integration of CMOS with your current monitoring pipelines can be achieved through configuration of existing Prometheus, Alertmanager, and Grafana instances
```

Comment:
Sure. Should
> While CMOS is designed as an all-in-one stack, it is also possible to deploy all its components separately. This tutorial will show you how to configure and deploy each component.
be simplified too?

Reply:
Any suggestions on simplifications?

Reply:
I'd say write it in the active voice is a good start - so rather than `Integration can be achieved through configuration`, something like `You can integrate CMOS with your existing Prometheus, Alertmanager, and Grafana monitoring system`. Something like that?

Reply:
Sounds good.